content/docs/command-reference/import.md

```diff
@@ -91,6 +112,9 @@ from the source repo.
   > [Importing and updating fixed revisions](#example-importing-and-updating-fixed-revisions)
   > example below).
 
+- `-s`, `--store` - [enables remote storage](#enabling-remote-storage) on for
+  the import, so it can be pushed (e.g. in case the original source is lost).
```

Comment:
```suggestion
- `-s`, `--store` - [enables remote storage](#enabling-remote-storage) for the
  import, so it can be pushed (e.g. in case the original source is lost).
```